src/utils/currencies.ts

```diff
@@ -287,6 +594,36 @@ export const FIAT_OPTIONS: Currency[] = [
         hasSymbol: "₫",
         maxFractionalDigits: 0
     },
+    {
+        label: "Central African CFA Franc BEAC XAF",
+        value: "XAF",
+        hasSymbol: "CFA Franc BEAC",
+        maxFractionalDigits: 2
+    },
```

Comment:
For the African Franc I think they should be denoted like this.
```suggestion
    {
        label: "Central African CFA Franc XAF",
        value: "XAF",
        hasSymbol: "F.CFA",
        maxFractionalDigits: 2
    },
```